### ISSUE TITLE
Any plan to add android_kotlin_binary and android_kotlin_library rules?

### ISSUE THREAD

**Issue by fishy** [pubref/rules_kotlin#5]
The current kotlin_library rule seems to be limited to pure kotlin library, I cannot find a way to use it to import Android libraries into kotlin (something like: https://kotlinlang.org/docs/tutorials/kotlin-android.html)

**Comment by pcj** [comment]
Hi @fishy, great idea.  I'd certainly like to see this, but I may not have the spare cycles to do it right now.

**Comment by lswith** [comment]
I'd love to see this feature as well.

**Comment by fishy** [comment]
Now that https://github.com/pubref/rules_kotlin/pull/32 is merged, I guess we can close this issue? :)

**Comment by pcj** [comment]
Agree.  

Closed by #32.

**Comment by edenman** [comment]
Should I open a separate issue for the `kotlin_android_binary` improvement?  Understood if it's not a priority right now, but I'd love to switch over to building my app with Bazel in CI :)

**Comment by fishy** [comment]
@edenman Is having two rules in your BUILD file, one `android_binary` and one `kotlin_android_library` not an option for your CI system? It should be trivial to add `kotlin_android_binary` with existing `kotlin_android_library` macro, I just don't feel it's very useful or needed.

**Comment by pcj** [comment]
@edenman See https://github.com/fishy/notifbot/blob/master/android/BUILD for example

**Comment by edenman** [comment]
Oh nice, thanks for the pointer!  Two rules is totally fine, I just didn't realize that would work.  Now I'm just blocked on #15, can't wait to give it a try once that's resolved!
